test/unit/spec/narrative_core/upload/uploadTour-spec.js

```diff
@@ -0,0 +1,141 @@
+                    user: fakeUser,
+                    idents: [
+                        {
+                            provider: 'Google',
+                            provusername: 'notAUser@google.com',
+                        },
+                        {
+                            provider: 'Globus',
+                            provusername: 'notauser@globusid.org',
+                        },
+                    ],
+                }),
+            });
+            jasmine.Ajax.stubRequest(new RegExp(`${stagingServiceUrl}/list/`)).andReturn({
+                status: 200,
+                statusText: 'success',
+                contentType: 'text/plain',
+                responseHeaders: '',
+                responseText: JSON.stringify([
+                    {
+                        name: 'test_folder',
+                        path: fakeUser + '/test_folder',
+                        mtime: 1532738637499,
+                        size: 34,
+                        isFolder: true,
+                    },
+                    {
+                        name: 'file_list.txt',
+                        path: fakeUser + '/test_folder/file_list.txt',
+                        mtime: 1532738637555,
+                        size: 49233,
+                        source: 'KBase upload',
+                        isFolder: false,
+                    },
+                ]),
+            });
+            const mappings = {
+                mappings: [null, null],
+            };
+            jasmine.Ajax.stubRequest(
+                new RegExp(`${stagingServiceUrl}/importer_mappings/`)
+            ).andReturn({
+                status: 200,
+                statusText: 'success',
+                contentType: 'text/plain',
+                responseHeaders: '',
+                responseText: JSON.stringify(mappings),
+            });
+
+            Jupyter.narrative = {
+                userId: fakeUser,
+                getAuthToken: () => {
+                    return 'fakeToken';
+                },
+                showDataOverlay: () => {},
+                hideOverlay: () => {},
+            };
+        });
+
+        afterEach(() => {
+            jasmine.Ajax.uninstall();
+        });
+
+        afterAll(() => {
+            TestUtil.clearRuntime();
+        });
+
+        it('Loaded the Tour module', () => {
+            expect(UploadTour).toEqual(jasmine.any(Object));
+            expect(UploadTour.Tour).toEqual(jasmine.any(Function));
+        });
+
+        it('Can be instantiated', () => {
+            const tourInstance = new UploadTour.Tour($('div'));
+            expect(tourInstance).not.toBe(null);
+        });
+
+        it('Can be started', () => {
+            const tourInstance = new UploadTour.Tour($('div'));
+            tourInstance.start();
+            expect(document.querySelector('.kb-tour')).toBeTruthy();
+            tourInstance.stop();
+        });
+
+        xit('Can run with the Globus step', async () => {
```

Comment:
This is the set of steps that should, in theory, test flowing through the tour. It's hanging up on a mix of getting the staging area instantiated (which itself is weird), and waiting on each popover to appear (which is also weird, and possibly time-dependent).

In the interest of project time, IMO, this should be left to user-testing for now.